tests/container/vertx/src/test/java/org/hawkular/apm/tests/client/vertx/ClientVertxEventBusTest.java

```diff
@@ -72,14 +73,6 @@ public void handle(AsyncResult<Message<String>> message) {
             }
         });
 
-        try {
-            synchronized (this) {
-                wait(2000);
-            }
-        } catch (Exception e) {
-            e.printStackTrace();
-        }
-
         mc.unregister();
```

Comment:
Haven't run the build yet, so may work fine - but just wondering whether if the consumer is unregistered before the completion handler fires (async), then no message would be sent on the bus. I think that is why I had to introduce the delays.

Maybe some synchronization could be performed, to wait until the message has been sent and reply received?